pkg/validator/types.go

```diff
@@ -2,6 +2,10 @@ package validator
 
 import (
 	"github.com/openservicemesh/osm/pkg/logger"
+	"github.com/pkg/errors"
 )
 
 var log = logger.New(ValidatorWebhookSvc)
+
+var ErrIngressBackendDuplicateBackends = errors.New("error: duplicate backends detected")
+var ErrUpstreamTrafficSettingDuplicateHosts = errors.New("error: duplicate hosts detected")
```

Comment:
```suggestion
var (
  ErrIngressBackendDuplicateBackends      = errors.New("error: duplicate backends detected")
  ErrUpstreamTrafficSettingDuplicateHosts = errors.New("error: duplicate hosts detected")
)
```

Reply:
In most errors we create we don't use the format "error: \<err msg\>". Instead, we do something like the following `var ErrCouldNotOpen = errors.New("could not open")`

Reply:
Ah, thanks for the heads up! Fixed